sprokit/processes/bindings/python/kwiver/kwiver_process.py

```diff
@@ -181,8 +181,12 @@ def __init__(self, conf):
 
         self.add_type_trait("homography_src_to_ref", "kwiver:s2r_homography")
         self.add_type_trait("homography_ref_to_src", "kwiver:r2s_homography")
-        self.add_type_trait("image_file_name", "kwiver:image_file_name")
-        self.add_type_trait("video_file_name", "kwiver:video_file_name")
+        self.add_type_trait("image_file_name", "kwiver:image_file_name",
+                            datum.Datum.get_image_file_name,
+                            datum.new_string)
```

Comment:
You could just make the "get" call get_string since that is the underlying data type. That may be useful for other items that are implemented as a string.